Content.Server/Power/EntitySystems/PowerReceiverSystem.cs

```diff
@@ -71,7 +71,8 @@ private void OnProviderShutdown(EntityUid uid, ApcPowerProviderComponent compone
         {
             foreach (var receiver in component.LinkedReceivers)
             {
-                receiver.NetworkLoad.LinkedNetwork = default;
+                var comp = Comp<ApcPowerReceiverComponent>(receiver);
+                comp.NetworkLoad.LinkedNetwork = default;
```

Comment:
Receivers can be shutdown first and there's no guarantee as far as I read this code.